### ISSUE TITLE
"Xamarin Forms" Checkbox gets the focus twice while Tabbing from "Ios/Android" to Create Workbook button.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#327]
### Pre-Requisite:
Follow the instructions for installing the app from 
https://docs.microsoft.com/en-us/xamarin/tools/workbooks/install?tabs=windows 

### Steps to Reproduce

1. Launch Xamarin Workbooks application.
2. Select the working platform to "Ios/Android".
3. Navigate to "Xamarin Forms" checkbox using Tab and then to "Create workbook" button from "Ios/Android"

### Expected Behavior
"Xamarin Forms" checkbox should get focus only once.
### Actual Behavior
"Xamarin Forms" Checkbox gets the focus twice while Tabbing from "Ios/Android" to Create Workbook button.
### Note:
On the first time it sets focus on "Xamarin Forms" , if user tries to check the checkbox , it will not be performed. Only on second time focus user can check the checkbox.

### User Impact
People with visual impairments can become disoriented when keyboard navigation takes focus somewhere unexpected or when they cannot easily find the content surrounding an interactive element.
### Environment
Windows : Version 1803 (OS Build : 17134.48)
Xamarin Version : 1.4(master/b3f92f9)
### Screenshots:
![issue 3](https://user-images.githubusercontent.com/22956130/40481672-beba12de-5f6f-11e8-9221-04fb1f6a8f27.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
